src/Appwrite/Platform/Tasks/ScheduleExecutions.php

```
@@ -28,17 +26,11 @@ public static function getCollectionId(): string
         return 'executions';
     }
 
-    protected function enqueueResources(Group $pools, Database $dbForPlatform, callable $getProjectDB): void
+    protected function enqueueResources(Database $dbForPlatform, callable $getProjectDB): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$getProjectDB'. (Unused Code Rules)

(UnusedFormalParameter)